nixos/modules/hardware/wooting.nix

```
@@ -1,7 +1,6 @@
 { config, lib, pkgs, ... }:
 {
-  options.hardware.wooting.enable = lib.mkEnableOption ''support for Wooting keyboards.
-    Note that users must be in the "input" group for udev rules to apply'';
+  options.hardware.wooting.enable = lib.mkEnableOption "Support for Wooting keyboards.";
```

```suggestion
  options.hardware.wooting.enable = lib.mkEnableOption "support for Wooting keyboards";
```
`mkEnableOption` makes the sentence automatically

https://github.com/NixOS/nixpkgs/blob/1b0e57933c5cc61ebae72e4d977809c8979f4a7e/lib/options.nix#L98-L105

---

Goodf point, I fixed it